### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Glebatikot**
- [DEMO LINK](https://Glebatikot.github.io/layout_catalog_grid/)

### CODE REVIEW

**Comment by budanov20** [comment]
добре

File: src/styles/_catalog.scss [hunk 1]
```diff
@@ -1,9 +1,24 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
```
Comment (VETALEK) [inline src/styles/_catalog.scss:3]: this rule never works, cause you covered all the scenarios with media queries 😭

Comment (VETALEK) [reply src/styles/_catalog.scss:3]: so you don't need it

**Review by VETALEK — CHANGES_REQUESTED** [review]
Nice 👌

**Review by VETALEK — APPROVED** [review]
🔥

**Pull request closed without merging**